package.json

```diff
@@ -1,95 +1,49 @@
-    "start": "ember serve",
-    "test": "npm-run-all lint test:*",
-    "test:ember": "ember test",
-    "test:ember-compatibility": "ember try:each"
-  },
-  "dependencies": {
-    "@embroider/macros": "^1.0.0",
-    "ember-cli-babel": "^7.26.11",
-    "ember-modifier-manager-polyfill": "^1.2.0"
+    "lint:addon": "yarn workspace @ember/render-modifiers run lint",
+    "lint:test-app": "yarn workspace test-app run lint",
+    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:fix:*",
+    "lint:fix:addon": "yarn workspace @ember/render-modifiers run lint:fix",
+    "lint:fix:test-app": "yarn workspace test-app run lint:fix",
+    "test": "npm-run-all --aggregate-output --continue-on-error --parallel \"test:!(watch)\"",
+    "test:watch": "npm-run-all --aggregate-output --continue-on-error --parallel test:watch:*",
+    "test:test-app": "yarn workspace test-app run test",
+    "test:watch:test-app": "yarn workspace test-app run test:watch",
+    "test:watch:addon": "yarn workspace @ember/render-modifiers run start",
+    "release": "release-it"
+  },
+  "volta": {
+    "node": "16.14.2",
+    "yarn": "1.22.17"
   },
   "devDependencies": {
-    "@ember/optional-features": "^2.0.0",
-    "@ember/test-helpers": "^2.6.0",
-    "@embroider/test-setup": "^1.0.0",
-    "@embroider/util": "^1.0.0",
-    "@glimmer/component": "^1.0.4",
-    "@glimmer/tracking": "^1.0.4",
-    "babel-eslint": "^10.1.0",
-    "ember-auto-import": "^2.2.4",
-    "ember-cli": "~4.1.0",
-    "ember-cli-dependency-checker": "^3.2.0",
-    "ember-cli-htmlbars": "^6.0.1",
-    "ember-cli-inject-live-reload": "^2.1.0",
-    "ember-cli-terser": "^4.0.2",
-    "ember-disable-prototype-extensions": "^1.1.3",
-    "ember-load-initializers": "^2.1.2",
-    "ember-page-title": "^7.0.0",
-    "ember-qunit": "^5.1.5",
-    "ember-resolver": "^8.0.3",
-    "ember-source": "~4.1.0",
-    "ember-source-channel-url": "^3.0.0",
-    "ember-template-lint": "^3.15.0",
-    "ember-try": "^2.0.0",
-    "eslint": "^7.32.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-ember": "^10.5.8",
-    "eslint-plugin-node": "^11.1.0",
-    "eslint-plugin-prettier": "^4.0.0",
-    "eslint-plugin-qunit": "^7.2.0",
-    "loader.js": "^4.7.0",
     "npm-run-all": "^4.1.5",
-    "prettier": "^2.5.1",
-    "qunit": "^2.17.2",
-    "qunit-dom": "^2.0.0",
-    "release-it": "^14.11.6",
-    "release-it-lerna-changelog": "^3.1.0",
-    "webpack": "^5.65.0"
-  },
-  "peerDependencies": {
-    "ember-source": "^3.8 || 4"
-  },
-  "engines": {
-    "node": "12.* || 14.* || >= 16"
-  },
-  "publishConfig": {
-    "access": "public",
-    "registry": "https://registry.npmjs.org"
-  },
-  "ember": {
-    "edition": "octane"
+    "release-it": "^14.0.0",
+    "release-it-lerna-changelog": "^4.0.1",
+    "release-it-yarn-workspaces": "^2.0.1"
   },
-  "ember-addon": {
-    "configPath": "tests/dummy/config"
+  "resolutions": {
+    "**/@glimmer/validator": "0.65.4"
```

Comment:
this is fishy, I don't think this is needed

Reply:
I agree, this was an attempt to fix `@glimmer/validator` issue. cf https://github.com/emberjs/ember-render-modifiers/pull/66#issuecomment-1277354436